electricitymap/contrib/config/co2eq_parameters.py

```diff
@@ -0,0 +1,60 @@
+"""Contains a function to make co2eq parameter dicts from
+config read from defaults.yaml and zones/*.yaml.
+"""
+
+from copy import deepcopy
+from typing import Dict
+
+
+def generate_co2eq_parameters(defaults, zones_config):
```

Comment:
It would be great if this had a return signature.

Reply:
Good point, thanks

Reply:
Done.